react/Viewer/Panel/ActionMenuWrapper.jsx

```diff
@@ -12,13 +12,23 @@ import ActionMenu, { ActionMenuItem } from '../../ActionMenu'
 import Typography from '../../Typography'
 import useBreakpoints from '../../hooks/useBreakpoints'
 import { useI18n } from '../../I18n'
+import useViewerSnackbar from '../snackbar/useViewerSnackbar'
 
 const ActionMenuWrapper = forwardRef(({ onClose, value }, ref) => {
   const { isMobile } = useBreakpoints()
   const { t } = useI18n()
+  const { showViewerSnackbar } = useViewerSnackbar()
 
   const handleCopy = () => {
-    navigator?.clipboard?.writeText(value)
+    if (navigator?.clipboard) {
+      navigator.clipboard.writeText(value)
+      showViewerSnackbar(
+        'secondary',
+        t(`Viewer.snackbar.copiedToClipboard.success`)
```

Comment:
This message doesn't appear on mobile, is it wanted?

Reply:
Non, problème résolu en désactivant le `Portal` du `BottomSheet` 👍